EasyPost/Shipment.cs

```diff
@@ -128,6 +128,18 @@ public class Shipment : Resource {
             rates = request.Execute<Shipment>().rates;
         }
 
+        public List<SmartRate> GetSmartRates()
```

Comment:
@jontsai the interface on the other CLs was `GetSmartrates`, with the `r` being lowercased. I'm assuming we want to follow that here?

Reply:
renamed to GetSmartrates..